app/utils/JSONHelper.ts

```diff
@@ -40,3 +40,23 @@ export function intToString(jsonStruct: string, key: string) {
         `"${key}":"$1"`
     );
 }
+
+// Given any object/array, turn all bigint fields to strings;
```

Comment:
Use
```
/**  */
```
when documenting functions.

Reply:
removed the function as it didn't work correctly.